routes/entryRoutes.js

```diff
@@ -2,21 +2,17 @@ const express = require('express');
 const router = express.Router();
 
 const {
-  addToFavorites,
   createEntry,
   deleteEntry,
   editEntry,
   getAllEntries,
   getEntry,
-  getFavorites,
```

Comment:
In future improvements, remember to getEmotions "Happy, Angry and Sad".

Reply:
@loitijoseph 🤔 not following, expound a bit more

Reply:
@rmoswela there was an idea we had to add some emotion to each diary entry. How you felt that day. 

@loitijoseph it's something we can add to the Entry Schema and it will be returned along with all the data.

Reply:
@ChinyangaTL

Ok cool, that will be awesome.

Reply:
@rmoswela did you get it now Sir?